lnbits/core/migrations.py

```diff
@@ -513,3 +513,30 @@ async def m019_balances_view_based_on_wallets(db):
         GROUP BY apipayments.wallet
     """
     )
+
+
+async def m020_add_success_failed_to_apipayments(db):
+    """
+    Adds success and failed columns to apipayments.
+    """
+    await db.execute("ALTER TABLE apipayments ADD COLUMN status TEXT DEFAULT 'pending'")
+    #  set all not pending to success true, failed payments were deleted until now
+    await db.execute("UPDATE apipayments SET status = 'success' WHERE NOT pending")
```

Comment:
`state` vs `status`
the class is `PaymentState` (due to name conflict), but the column and variables are `status`